### DISCUSSION TITLE
如何将中英文标题改成三行显示

### DISCUSSION THREAD

**Discussion by jinziyi528** [TheNetAdmin/zjuthesis#331] [General]
因为目前标定的中英文只支持两行显示，如何支持三行的显示

**Comment by genema** [comment]
### 此处只介绍中文 cover 修改三行英文 title的情况，其他情况类似，可自行扩展。
**./page/graduate/cover_chn.tex** ：
*start from line 78*
       ` {`
            `% TitleEngLines == 3  % 2   % 这里直接覆盖了两行标题的实现`
            `英文论文题目：& \zihao{4} \uline{\hfill \TitleEngLineOne{} \hfill} \\`
           ` ~            & \zihao{4} \uline{\hfill \TitleEngLineTwo{} \hfill} \\`
            `~             & \zihao{4} \uline{\hfill \TitleEngLineThree{} \hfill} \\  % **增加一行**`
       ` }`

**./config/format/general/title.tex** (其他 major 的模板也是共用 general的 title config)
增加定义
`\newcommand{\TitleEngLineThree}{~}`
`\newcommand{\titleengthreelines}[3]{`
`    \renewcommand{\TitleEngLines}{3}`
`    \renewcommand{\TitleEngLineOne}{{#1}}`
`    \renewcommand{\TitleEngLineTwo}{{#2}}`
`    \renewcommand{\TitleEngLineThree}{{#3}}`
`}`
然后在主文件中按照两行标题的方式写就行：
`\titleengthreelines{第一行}{第二行}{第三行}`
